views-bom/build.gradle

```diff
@@ -1,15 +1,12 @@
-dependencies {
-    constraints {
-        for (Project p : rootProject.subprojects) {
-            if (p.name == "views-bom" || p.name == 'views') continue
-            api "$p.group:micronaut-$p.name:$p.version"
-        }
-
-        runtime "org.freemarker:freemarker:$freemarkerVersion"
-        runtime "com.github.jknack:handlebars:$handlebarsVersion"
-        runtime "com.fizzed:rocker-runtime:$rockerVersion"
-        runtime "com.google.template:soy:$soyVersion"
-        runtime "org.thymeleaf:thymeleaf:$thymeleafVersion"
-        runtime "org.apache.velocity:velocity-engine-core:$velocityVersion"
-    }
+plugins {
+    id "io.micronaut.build.internal.bom"
 }
+
+dependencies {
+    api("org.freemarker:freemarker:$freemarkerVersion")
+    api("com.github.jknack:handlebars:$handlebarsVersion")
+    api("com.fizzed:rocker-runtime:$rockerVersion")
+    api("com.google.template:soy:$soyVersion")
+    api("org.thymeleaf:thymeleaf:$thymeleafVersion")
+    api("org.apache.velocity:velocity-engine-core:$velocityVersion")
+}
```

Comment:
@sdelamo @melix Shouldn't `constraints {` have been preserved here?

I just gave `micronaut-bom:3.2.2` a try (declares `micronaut-views-bom:3.0.2`) and this leads to *much* fatter jars.

Reply:
Yes, they should be in `constraints`, good catch!

Reply:
Fixed via https://github.com/micronaut-projects/micronaut-views/pull/331